src/Hng.Domain/Entities/EmailTemplate.cs

```diff
@@ -1,7 +1,12 @@
+using System.ComponentModel.DataAnnotations;
+using Microsoft.EntityFrameworkCore;
+
 namespace Hng.Domain.Entities;
 
+[Index(nameof(Name), IsUnique = true)]
```

Comment:
How come this is here and not inside the class above name?

Reply:
Index attribute takes classes as arguments.